tests/integration/logical/test_snapshot.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test 'snapshot'.
+"""
+
+import time
+import unittest
+
+from stratis_cli._errors import StratisCliDbusLookupError
+
+from .._misc import _device_list
+from .._misc import RUNNER
+from .._misc import Service
+
+
+_DEVICE_STRATEGY = _device_list(1)
+
+
+class SnapshotTestCase(unittest.TestCase):
+    """
+    Test creating a snapshot of a filesystem in a pool.  In this case
+    the snapshot should be created and no error rasied.
+    """
+    _MENU = ['--propagate', 'filesystem', 'snapshot']
+    _POOLNAME = 'deadpool'
+    _SNAPNAME = 'snapfs'
+    _FSNAME = 'fs'
+
+    def setUp(self):
+        """
+        Start the stratisd daemon with the simulator.
+        """
+        self._service = Service()
+        self._service.setUp()
+        time.sleep(1)
+        command_line = \
```

Comment:
There's probably some oversights in some examples, but better to be less convoluted with lists, like:

```['pool', 'create', self._POOLNAME, _DEVICE_STRATEGY.example()]```